infrastructure/index.ts

```diff
@@ -220,15 +295,166 @@ const baseCacheBehavior = {
+            }
+        }
+    );
+    bundleOrigins.push(
+        {
+            originId: bundlesBucket.arn,
+            domainName: bundlesBucket.websiteEndpoint,
+            customOriginConfig: {
+                originProtocolPolicy: "http-only",
+                httpPort: 80,
+                httpsPort: 443,
+                originSslProtocols: ["TLSv1.2"],
+            }
+        }
+    );
+    registryBehaviors.push(
+        {
+            ...baseCacheBehavior,
+            targetOriginId: registryBucket.arn,
+            pathPattern: "/registry/*",
+            defaultTtl: oneHour,
+            maxTtl: oneHour,
+            forwardedValues: {
+                cookies: {
+                    forward: "none",
+                },
+                queryString: false,
+                headers: [
+                    "Origin",
+                    "Access-Control-Request-Headers",
+                    "Access-Control-Request-Method",
+                ],
+            },
+        },
+        {
+            ...baseCacheBehavior,
+            targetOriginId: registryBucket.arn,
+            pathPattern: "/docs/cli/commands/*",
+            defaultTtl: oneHour,
+            maxTtl: oneHour,
+            forwardedValues: {
+                cookies: {
+                    forward: "none",
+                },
+                queryString: false,
+                headers: [
+                    "Origin",
+                    "Access-Control-Request-Headers",
+                    "Access-Control-Request-Method",
+                ],
+            },
+        },
+        {
+            ...baseCacheBehavior,
+            targetOriginId: registryBucket.arn,
+            pathPattern: "/docs/install/*",
+            defaultTtl: oneHour,
+            maxTtl: oneHour,
+            forwardedValues: {
+                cookies: {
+                    forward: "none",
+                },
+                queryString: false,
+                headers: [
+                    "Origin",
+                    "Access-Control-Request-Headers",
+                    "Access-Control-Request-Method",
+                ],
+            },
+        },
+        {
+            ...baseCacheBehavior,
+            targetOriginId: registryBucket.arn,
+            pathPattern: "/docs/reference/pkg/*",
+            defaultTtl: oneHour,
+            maxTtl: oneHour,
+            forwardedValues: {
+                cookies: {
+                    forward: "none",
+                },
+                queryString: false,
+                headers: [
+                    "Origin",
+                    "Access-Control-Request-Headers",
+                    "Access-Control-Request-Method",
+                ],
+            },
+        },
+    )
+    bundleBehaviors.push(
+        {
+            ...baseCacheBehavior,
+            targetOriginId: bundlesBucket.arn,
+            pathPattern: "/css/*",
+            defaultTtl: oneHour,
+            maxTtl: oneHour,
```

Comment:
Should we keep js/css assets around longer in the cache? Setting it to one hour just feels like we might run into issues where HTML is going to be requesting a version that is no longer available.

Reply:
yeah we could increase it, but the previous versions of the assets will continue to live in the bundles bucket until we come up with a way to delete the old assets. Right now i'm just pushing in the new bundle files to that bucket when they get built, so the old ones are not being deleted as of now (which may not be a bad thing) so they will technically still be available. I'm thinking if we want we could just have a job that deletes assets that are over a month old or something like that if we want.